index.html

```diff
@@ -19,17 +19,17 @@ <h1 class="text-center" >
 		For numbers which are multiples of both three and five print “<b class="text-red">FizzBuzz</b>”.
 	</p>
 	<div class="text-center  " id="main-button">
-	   <button id="play-button"><i style="font-size:24px" class="fa">&#xf04b;</i> Play</button>
+	   <button id="play-button" ><i style="font-size:24px" class="fa">&#xf04b;</i> <span>Play</span></button>
```

Comment:
Let's remove this span, it's not needed anymore
```suggestion
	   <button id="play-button" ><i style="font-size:24px" class="fa">&#xf04b;</i> Play</button>
```